packages/core/admin/admin/src/content-manager/utils/schema.js

```diff
@@ -214,6 +219,7 @@ const createYupSchemaAttribute = (type, validations, options) => {
   if (type === 'json') {
     schema = yup
       .mixed(errorsTrads.json)
+      .transform((value) => (options.formatJsonBeforeValidate ? JSON.stringify(value) : value))
```

Comment:
Could we avoid the option with a typeof check?

Reply:
That's true. Also it looks like it is impossible to save a draft with invalid JSON. So if we have an object then the input must be valid, right? We only need to run JSON format validation in the edit view since it is impossible to have invalid JSON in the ListView. 

So would it make sense to get rid of this transform and just return true `if typeof === object || null`? It sounds like the only validation error we should be reporting in the list view for JSON is `required`.